llmstack/cli.py

```diff
@@ -3,16 +3,18 @@
 import secrets
 import subprocess
 import sys
+import argparse
 from collections import defaultdict
 
 import docker
 import toml
 
 
-def run_django_command(command: list[str] = ['manage.py', 'runserver']):
+def run_django_command(command: list[str] = ["manage.py", "runserver"]):
```

Comment:
Please undo all the linter changes related to changing single quotes to double quotes.